README.md

```diff
@@ -256,6 +252,13 @@ sudo apt-get install libudev-dev
 ```
 
 
+# To build a debug version with error outputs use:
+
+```sh
+$ npm run rebuild --debug
+```
```

Comment:
Out of place since beginning, https://github.com/MadLittleMods/node-usb-detection/commit/984df633c1fc66f925acd115628f90a498bca579

Not sure either